man/org.freedesktop.sysupdate1.xml

```diff
@@ -3,7 +3,7 @@
   "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd" >
 <!-- SPDX-License-Identifier: LGPL-2.1-or-later -->
 
-<refentry id="org.freedesktop.sysupdate1" conditional='ENABLE_SYSUPDATE'
+<refentry id="org.freedesktop.sysupdate1" conditional='ENABLE_SYSUPDATE ENABLE_SYSUPDATED'
```

Comment:
Not a big matter, but when `ENABLE_SYSUPDATED` is true, `ENABLE_SYSUPDATE` is always true.
Hence, `ENABLE_SYSUPDATE` can be dropped.